### PR TITLE
Milestone6: views

### PR DESCRIPTION
**Pull request by Favourezeugwa**
## I Implemented the following in this milestone;
- Implemented the design from the sneak peek wireframes.
- Used methods that were created in the models

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1 +1,28 @@
...
+        <h5><%= link_to post.title, user_post_path(@user, post) %></h5>
+        <p><%= post.text %></p>
+        <div class='like-comment'>
+          <li>Comments: <%= post.comments_counter %>, </li>
+          <li>Likes: <%= post.likes_counter %></li>
+        </div>
+      </div>
+      <ul class='post-comment'>
+        <% post.comments.each do |comment| %>
+          <li><%= comment.users.name %>: <%= comment.text %></li>
```
Comment (Ismailco) [inline app/views/posts/index.html.erb:23]: - [ ] You should use `post.recent_comments` to render the 5 most recent comments in the posts index page, kindly implement this so only the last 5 comments are rendred here

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @Favourezeugwa, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb (hunk 1 above)
Comment (Favourezeugwa) [reply app/views/posts/index.html.erb:23]: okay @Ismailco. I will fix it

**Review by KDlamini — APPROVED** [review]
### Hi @Favourezeugwa 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the required changes. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

## HIGHLIGHTS 🌟 

✔️ Required views implemented
✔️ Commit messages are meaningful
✔️ No linter errors

# Status: Approved ✔️ 

giphy-tumblr

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Favourezeugwa** [comment]
Thanks @KDlamini

**Pull request merged**